src/spikeinterface/core/tests/test_binaryrecordingextractor.py

```diff
@@ -33,7 +33,7 @@ def test_BinaryRecordingExtractor(create_cache_folder):
 
 def test_round_trip(tmp_path):
     num_channels = 10
-    num_samples = 50
+    num_samples = 500
```

Comment:
why ?
did the new change trigered a new errors with veru small signals ?

Reply:
because the values in the tests ar start_frame 200/end_frame 400, so I think this was wrong